README.md

```diff
@@ -124,16 +124,51 @@ Use [Homebrew] to install the packages:
 brew install sdl2 ffmpeg
 
 # client build dependencies
-brew install gcc pkg-config meson
+brew install pkg-config meson
```

Comment:
A C compiler is available by default on MacOS?

Reply:
If you have brew it sure is (clang)